src/ocf_freelist.c

```diff
@@ -365,7 +365,7 @@ void ocf_freelist_put_cache_line(ocf_freelist_t freelist,
 	env_put_execution_context(ctx);
 }
 
-ocf_freelist_t ocf_freelist_init(struct ocf_cache *cache)
+int ocf_freelist_init(ocf_freelist_t *flst, struct ocf_cache *cache)
```

Comment:
By convention when we return by pointer we typically name local pointer something like `tmp_freelist` and the function argument pointer `freelist`.